docs/develop/go/versioning.mdx

```diff
@@ -179,31 +179,23 @@ But we don't want to fail on that change, so we check only the function name par
 
 :::caution
 
-Worker Versioning is currently in Pre-release, and backwards-incompatible changes are coming to the Worker Versioning APIs. For now, you need to provide dynamic configuration parameters to your Temporal Service to enable Worker Versioning:
+Worker Versioning is currently in Pre-release.
 
-```
-temporal server start-dev \
-   --dynamic-config-value frontend.workerVersioningDataAPIs=true \
-   --dynamic-config-value frontend.workerVersioningWorkflowAPIs=true \
-   --dynamic-config-value worker.buildIdScavengerEnabled=true
-```
+See the [Pre-release README.md](https://github.com/temporalio/temporal/blob/main/docs/worker-versioning.md)
+for full information.
 
 :::
 
-To use Worker Versioning in Go, you need to do the following:
-
-1. Determine and assign a Build ID to your built Worker code, and opt in to versioning.
-2. Tell the Task Queue your Worker is listening on about that Build ID, and whether its compatible with an existing Build ID.
+To use Worker Versioning for Go workers, you need to determine and assign a Build ID to your Worker code, and opt in to Worker Versioning.
```

Comment:
Wording: To add Worker Versioning, you must create and use a unique Build ID in your Worker Code, and opt into Worker Versioning features in your `WorkerOptions` settings.

Seems to be missing the bit about what a Build ID is _before_ assignment.

e.g. 
```
Each Build ID identifies the Worker capabilities available in this deployment.
BuildIDs strings might use semantic versioning strings, hashes, or business descriptions.
```